test/arguments.ts

```diff
@@ -104,13 +107,13 @@ test('overrides `searchParams` from options', withServer, async (t, server, got)
 				test: 'wow'
 			},
 			cache: {
-				get(key) {
+				get(key: string) {
 					t.is(key, `cacheable-request:GET:${server.url}/?test=wow`);
 				},
-				set(key) {
+				set(key: string) {
 					t.is(key, `cacheable-request:GET:${server.url}/?test=wow`);
 				}
-			}
+			} as unknown as StorageAdapter
```

Comment:
Is this really needed? I think you just need to define the missing methods. The definition is:

```ts
interface Store<TValue> {
        get(key: string): TValue | Promise<TValue | undefined> | undefined;
        set(key: string, value: TValue, ttl?: number): any;
        delete(key: string): boolean | Promise<boolean>;
        clear(): void | Promise<void>;
    }
```

Reply:
This is resolved by implementing all methods.